base_layer/core/src/transactions/key_manager/interface.rs

```diff
@@ -58,13 +61,13 @@ pub enum TxoStage {
 #[derive(Clone, Copy, EnumIter)]
 pub enum TransactionKeyManagerBranch {
     DataEncryption = 0x00,
-    Alpha = 0x01,
-    MetadataEphemeralNonce = 0x02,
-    CommitmentMask = 0x03,
-    Nonce = 0x04,
-    KernelNonce = 0x05,
-    SenderOffset = 0x06,
-    SenderOffsetLedger = 0x07,
+    MetadataEphemeralNonce = 0x01,
```

Comment:
This looks like a `BREAKING CHANGE` here

Reply:
yeah its breaking change for ledger, not software, good catch